pymilvus/client/stub.py

```diff
@@ -42,7 +42,7 @@ def inner(self, *args, **kwargs):
     return inner
 
 
-def retry_on_rpc_failure(retry_times=10, wait=1):
+def retry_on_rpc_failure(retry_times=10, wait=1, retry_on_deadline=True):
```

Comment:
just rename retry_on_deadline to  `enable`, deafult  `enable=True`

Reply:
so what if we will distinguish the behavior of handling `UNAVAILABLE`?
maybe you mean `enable_on_deadline` and default `enable_on_deadline=True`?

Reply:
Add it（`enable_on_deadline`） when we really want to distinguish. Anyway, this `enable` to seems reasonable and will be reserved even if we add `enable_on_dealine`.